Dockerfile

```diff
@@ -1,18 +1,28 @@
+# syntax=docker/dockerfile:experimental
```

Comment:
What is the experimental part we are using?

Reply:
BuildKit is technically experimental syntax... Maybe. It works on GH Actions though, so /shrug?